### PR TITLE
{bio}[foss/2022a] HiC-Pro v3.1.0, hic-straw v1.3.1, iced v0.5.10, ... w/ R 4.2.1

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

dependencies for https://github.com/ay-lab/FitHiChIP (according to https://ay-lab.github.io/FitHiChIP/html/usage/installation.html#option-1-install-from-source-in-github)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/2c447940ea8855e5f5f9f79e5bd68b54 for a full test report.

File: easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2022a-R-4.2.1.eb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+easyblock = 'MakeCp'
+
+name = 'HiC-Pro'
+version = '3.1.0'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'http://nservant.github.io/HiC-Pro'
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2022a-R-4.2.1.eb:7]:
```suggestion
homepage = 'https://nservant.github.io/HiC-Pro'
```

File: easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2022a-R-4.2.1.eb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    ('Bowtie2', '2.4.5'),
+    ('Pysam', '0.19.1'),
+    ('SAMtools', '1.16.1'),
+    ('bx-python', '0.9.0'),
+    ('iced', '0.5.10'),
+]
+
+skipsteps = ['configure']
+
+prebuildopts = 'sed -i "s|PREFIX = |PREFIX = %(installdir)s|" config-install.txt && '
```
Comment (verdurin) [inline easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2022a-R-4.2.1.eb:28]: Could you add a quick summary of what's going on here, for the benefit of future maintainers?

Comment (verdurin) [reply easybuild/easyconfigs/h/HiC-Pro/HiC-Pro-3.1.0-foss-2022a-R-4.2.1.eb:28]: As a comment, I mean.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/5bcaf7b757f84584c7896f6c53fcd03e for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Pull request merged**